doc/examples/realtime_servo/realtime_servo_tutorial.rst

```diff
@@ -1,181 +1,196 @@
-Realtime Arm Servoing
-=====================
+Realtime Servo
+===============
 
-MoveIt Servo allows you to stream End Effector (EEF) velocity commands to your manipulator and have it execute them concurrently. This enables teleoperation via a wide range of input schemes, or for other autonomous software to control the robot - in visual servoing or closed loop position control for instance.
+MoveIt Servo facilitates realtime control of your robot arm.
 
-This tutorial shows how to use MoveIt Servo to send real-time servo commands to a ROS-enabled robot. Some nice features of the servo node are singularity handling and collision checking that prevents the operator from breaking the robot.
+Servo accepts any of the following types of commands:
```

Comment:
```suggestion
MoveIt Servo accepts any of the following types of commands:
```

This is true in a lot of other places as well. Try to say "MoveIt Servo" instead of just "Servo"

Reply:
meh